spec/expression.dd

```diff
@@ -1091,6 +1091,32 @@ $(GNAME CastExpression):
         }
         ---
 
+
+    $(P Casting a static array to another static array is done only if the
+        array lengths multiplied by the element sizes match; a mismatch
+        is illegal.
+        The cast is done as a type paint (aka a reinterpret cast).
+        The contents of the array are not changed.
+
+        ---
+        import core.stdc.stdio;
+
+        void main()
+        {
+            byte[16] b = 4;
+            int[4] ia = cast(int[4]) b;
+            foreach (i; ia)
+                printf("%x\n");
+            /* prints:
+               3030303
+               3030303
+               3030303
+               3030303
```

Comment:
It prints "4040404", doesn't it?

Reply:
Thanks! That's what I get for typing a working program in again. Fixed.

Reply:
Doesn't look like it. Forgot to push?

Reply:
see line 1106

Reply:
Ah.